Source/DataSources/BillboardVisualizer.js

```diff
@@ -162,6 +164,35 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
+     * @returns {BoundingSphereState} BoundingSphereState.DONE if the result contains the bounding sphere,
+     *                       BoundingSphereState.PENDING if the result is still being computed, or
+     *                       BoundingSphereState.FAILED if the entity has no visualization in the current scene.
+     */
+    BillboardVisualizer.prototype.getBoundingSphere = function(entity, result) {
```

Comment:
It looks like an implementation detail because it is a plug-in point (the `Visualizer` interface).  It's no different than `ImageryProvider` or `Datasource` interfaces.  Keep in mind that almost no one creates these visualizer classes directly (unless they are doing advanced stuff) so the fact that this method is documented is no different than the `BillboardVisualizer` class being documented as a whole.

Like I said, I'm fine with marking the entire Visualizer plugin point private if you want to; but if it is going to be public, then this needs to be public too.  It doesn't make sense for just this one function to be private.